site/guide/_manage-roles.qmd

```diff
@@ -0,0 +1,21 @@
+1. Under User Permissions, select **Roles**.
+
+1. Select the role for which you want to manage.
+
+1. Select **Edit Permissions** to edit its details. Within the role's details, you can:
```

Comment:
In general, we want to stay away from the nebulous "it" – simply state what "it" is. 

e.g. "Click **Edit Permissions** to add or remove permissions to or from that role." 

I would try and break this section down into the "assign permissions to roles" and "add users to roles" as these are actually separate tasks. You'll need to make sure it works with the training as well.